buyer.go

```diff
@@ -146,18 +153,64 @@ func newTicketPurchaser(cfg *config,
 		priceMode = AvgPriceDualMode
 	}
 
+	// Here we attempt to load purchased.csv from the webui dir.  This
+	// allows us to attempt to see if there have been previous ticketbuyer
+	// instances during the current stakediff window and reuse the
+	// previously tracked amount of tickets to purchase during that window.
+	f, err := os.OpenFile(filepath.Join(csvPath, csvPurchasedFn),
+		os.O_RDONLY|os.O_CREATE, 0600)
+	if err != nil {
+		log.Errorf("Error opening file: %v", err)
+		return nil, err
+	}
+	defer f.Close()
+
+	rdr := csv.NewReader(f)
+	rdr.Comma = ','
+	prevRecord := []string{}
+	for {
+		record, err := rdr.Read()
+		if err != nil {
+			if err == io.EOF {
+				break
+			}
+		}
+		prevRecord = record
+	}
+
+	// Attempt to parse last line in purchased.csv, then set previous amounts.
+	prevToBuyDiffPeriod := 0
+	lastHeight := 0
+	if len(prevRecord) >= 3 {
+		if prevRecord[1] == "RemainingToBuy" {
```

Comment:
Will the last line always be this in the CSV? We can depend on this for now, but if we modify this in the future it may not be the case. I'm not sure there's a better way to do this without just scanning the CSV for the latest block.